kurtosis-module/impl/testnet_verifier/testnet_verifier.go

```diff
@@ -0,0 +1,99 @@
+package testnet_verifier
+
+import (
+	"fmt"
+
+	"github.com/kurtosis-tech/eth2-merge-kurtosis-module/kurtosis-module/impl/module_io"
+	"github.com/kurtosis-tech/eth2-merge-kurtosis-module/kurtosis-module/impl/participant_network/cl"
+	"github.com/kurtosis-tech/eth2-merge-kurtosis-module/kurtosis-module/impl/participant_network/el"
+	"github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/enclaves"
+	"github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/services"
+	"github.com/kurtosis-tech/stacktrace"
+)
+
+// We use Docker exec commands to run the commands we need, so we override the default
+var entrypointArgs = []string{
+	"sleep",
+	"999999",
+}
+
+const (
+	imageName = "marioevz/merge-testnet-verifier:latest"
+	serviceId = "testnet-verifier"
+)
+
+func LaunchTestnetVerifier(params *module_io.ExecuteParams, enclaveCtx *enclaves.EnclaveContext, elClientCtxs []*el.ELClientContext, clClientCtxs []*cl.CLClientContext, ttd uint64) error {
```

Comment:
What if we call this `LaunchAsynchronousTestnetVerifier` to distinguish it from the synchronous case?